### ISSUE TITLE
Breaks swipe recognizers in the scroll view

### ISSUE THREAD

**Issue by Eugleo** [HarshilShah/DeckTransition#35]
My modal view controller is a collection view controller, in which some of the cells are custom slider controls (subclasses of UIControl). Probably because of the pan gesture recognizer, the controls don't work. I use the default beginTracking, etc., UIControl methods.

What'd be the best way to handle this problem?

**Comment by HarshilShah** [comment]
Could you possibly upload a demo project showing a test case? You could just add one of your custom controls that replicates the issue to the example app's modal.

I haven't experienced any issues myself, though I've never used Deck with custom UIControls other than buttons. Thanks!

**Comment by Eugleo** [comment]
Sure, [here is the whole Example Project](https://drive.google.com/file/d/0B9rLyNiJ2eePVm5Sb2c2NktxTHc/view?usp=sharing). I'm using CocoaPods. Also, I used SnapKit instead of normal frame-based layout or plain autolayout, because that's what I use in the original project.

**Comment by HarshilShah** [comment]
Thanks for the demo project, it helps a lot! I'll take a look

**Comment by HarshilShah** [comment]
Hmm, so I see your cells have delegate methods that print something whenever the slider is interacted with. I turned off Deck and presented with the default iOS style, and those delegates still print nothing. Maybe an issue somewhere else?

**Comment by Eugleo** [comment]
Oh, my bad. You'll need to make two changes: firstly change `func updateLayerFrames` from private no non-private, and then change the cellForItemAt method in ModalCollecionViewController to:

```
let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reuseIdentifier, for: indexPath) as! SliderCollectionViewCell
cell.slider.updateLayerFrames()
return cell
```

**Comment by HarshilShah** [comment]
Thanks for the info! Fixed in 1.0.4
